libraries/caffe-1.0.x/Dockerfile

```diff
@@ -0,0 +1,49 @@
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
The installation gets stuck:

```
➜  caffe docker build -f Dockerfile -t caffe-1.0.x-cuda10-ubuntu18.04 .
Sending build context to Docker daemon  28.67kB
Step 1/13 : FROM nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04
 ---> 074d6f612d7b
Step 2/13 : RUN apt-get -y update &&     apt-get install --no-install-recommends -y         build-essential         ca-certificates         checkinstall         libbz2-dev         libc6-dev         libffi-dev         libgdbm-dev         libncursesw5-dev         libreadline-gplv2-dev         libsqlite3-dev         libssl-dev         tk-dev         uuid-dev         wget &&     rm -rf /var/lib/apt/lists/*
 ---> Running in fe0629791c5e
Ign:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease
Ign:2 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease
Get:3 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release [564 B]
Get:4 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release [564 B]
Get:5 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release.gpg [801 B]
Get:6 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release.gpg [801 B]
Get:7 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
Get:8 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
Get:9 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Packages [85.5 kB]
Get:10 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Packages [9998 B]
Get:11 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [553 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages [13.5 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages [1344 kB]
Get:16 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [4169 B]
Get:17 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [5436 B]
Get:18 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [719 kB]
Get:19 http://archive.ubuntu.com/ubuntu bionic/multiverse amd64 Packages [186 kB]
Get:20 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages [11.3 MB]
Get:21 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [7239 B]
Get:22 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [10.8 kB]
Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [856 kB]
Get:24 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [1225 kB]
Get:25 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [3927 B]
Get:26 http://archive.ubuntu.com/ubuntu bionic-backports/main amd64 Packages [2496 B]
Fetched 16.9 MB in 3s (5918 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
build-essential is already the newest version (12.4ubuntu1).
build-essential set to manually installed.
ca-certificates is already the newest version (20180409).
libc6-dev is already the newest version (2.27-3ubuntu1).
libc6-dev set to manually installed.
The following additional packages will be installed:
  file fontconfig-config fonts-dejavu-core libbsd0 libexpat1 libexpat1-dev
  libfontconfig1 libfontconfig1-dev libfreetype6 libfreetype6-dev libglib2.0-0
  libice-dev libice6 libmagic-mgc libmagic1 libpng-dev libpng16-16 libpsl5
  libpthread-stubs0-dev libreadline5 libsm-dev libsm6 libsqlite3-0 libssl1.1
  libtcl8.6 libtinfo-dev libtk8.6 libx11-6 libx11-data libx11-dev libxau-dev
  libxau6 libxcb1 libxcb1-dev libxdmcp-dev libxdmcp6 libxext-dev libxext6
  libxft-dev libxft2 libxrender-dev libxrender1 libxss-dev libxss1 libxt-dev
  libxt6 multiarch-support pkg-config tcl tcl-dev tcl8.6 tcl8.6-dev tk tk8.6
  tk8.6-dev tzdata ucf x11-common x11proto-core-dev x11proto-dev
  x11proto-scrnsaver-dev x11proto-xext-dev xorg-sgml-doctools xtrans-dev
  zlib1g-dev
Suggested packages:
  gettext libice-doc ncurses-doc libsm-doc sqlite3-doc libssl-doc libxcb-doc
  libxext-doc libxt-doc tcl-doc tcl-tclreadline tcl8.6-doc tk-doc tk8.6-doc
Recommended packages:
  bzip2-doc libglib2.0-data shared-mime-info xdg-user-dirs libpng-tools
  publicsuffix libx11-doc xterm | x-terminal-emulator
The following NEW packages will be installed:
  checkinstall file fontconfig-config fonts-dejavu-core libbsd0 libbz2-dev
  libexpat1 libexpat1-dev libffi-dev libfontconfig1 libfontconfig1-dev
  libfreetype6 libfreetype6-dev libgdbm-dev libglib2.0-0 libice-dev libice6
  libmagic-mgc libmagic1 libncursesw5-dev libpng-dev libpng16-16 libpsl5
  libpthread-stubs0-dev libreadline-gplv2-dev libreadline5 libsm-dev libsm6
  libsqlite3-dev libssl-dev libtcl8.6 libtinfo-dev libtk8.6 libx11-6
  libx11-data libx11-dev libxau-dev libxau6 libxcb1 libxcb1-dev libxdmcp-dev
  libxdmcp6 libxext-dev libxext6 libxft-dev libxft2 libxrender-dev libxrender1
  libxss-dev libxss1 libxt-dev libxt6 multiarch-support pkg-config tcl tcl-dev
  tcl8.6 tcl8.6-dev tk tk-dev tk8.6 tk8.6-dev tzdata ucf uuid-dev wget
  x11-common x11proto-core-dev x11proto-dev x11proto-scrnsaver-dev
  x11proto-xext-dev xorg-sgml-doctools xtrans-dev zlib1g-dev
The following packages will be upgraded:
  libsqlite3-0 libssl1.1
2 upgraded, 74 newly installed, 0 to remove and 42 not upgraded.
Need to get 18.5 MB of archives.
After this operation, 71.1 MB of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 multiarch-support amd64 2.27-3ubuntu1 [6916 B]
Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxau6 amd64 1:1.0.8-1 [8376 B]
Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 libbsd0 amd64 0.8.7-1 [41.5 kB]
Get:4 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxdmcp6 amd64 1:1.1.2-3 [10.7 kB]
Get:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb1 amd64 1.13-2~ubuntu18.04 [45.5 kB]
Get:6 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libx11-data all 2:1.6.4-3ubuntu0.2 [113 kB]
Get:7 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libx11-6 amd64 2:1.6.4-3ubuntu0.2 [569 kB]
Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxext6 amd64 2:1.3.3-1 [29.4 kB]
Get:9 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 x11-common all 1:7.7+19ubuntu7.1 [22.5 kB]
Get:10 http://archive.ubuntu.com/ubuntu bionic/main amd64 libice6 amd64 2:1.0.9-2 [40.2 kB]
Get:11 http://archive.ubuntu.com/ubuntu bionic/main amd64 libsm6 amd64 2:1.2.2-1 [15.8 kB]
Get:12 http://archive.ubuntu.com/ubuntu bionic/main amd64 libexpat1 amd64 2.2.5-3 [80.2 kB]
Get:13 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpng16-16 amd64 1.6.34-1ubuntu0.18.04.2 [176 kB]
Get:14 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfreetype6 amd64 2.8.1-2ubuntu2 [335 kB]
Get:15 http://archive.ubuntu.com/ubuntu bionic/main amd64 ucf all 3.0038 [50.5 kB]
Get:16 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-dejavu-core all 2.37-1 [1041 kB]
Get:17 http://archive.ubuntu.com/ubuntu bionic/main amd64 fontconfig-config all 2.12.6-0ubuntu2 [55.8 kB]
Get:18 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfontconfig1 amd64 2.12.6-0ubuntu2 [137 kB]
Get:19 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxrender1 amd64 1:0.9.10-1 [18.7 kB]
Get:20 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxft2 amd64 2.3.2-1 [36.1 kB]
Get:21 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxss1 amd64 1:1.2.2-1 [8582 B]
Get:22 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic-mgc amd64 1:5.32-2ubuntu0.2 [184 kB]
Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic1 amd64 1:5.32-2ubuntu0.2 [68.5 kB]
Get:24 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 file amd64 1:5.32-2ubuntu0.2 [22.1 kB]
Get:25 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libglib2.0-0 amd64 2.56.4-0ubuntu0.18.04.3 [1169 kB]
Get:26 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libsqlite3-0 amd64 3.22.0-1ubuntu0.1 [497 kB]
Get:27 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libssl1.1 amd64 1.1.1-1ubuntu2.1~18.04.3 [1295 kB]
Get:28 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2019a-0ubuntu0.18.04 [189 kB]
Get:29 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpsl5 amd64 0.19.1-5build1 [41.8 kB]
Get:30 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 wget amd64 1.19.4-1ubuntu2.2 [316 kB]
Get:31 http://archive.ubuntu.com/ubuntu bionic/universe amd64 checkinstall amd64 1.6.2-4ubuntu2 [97.1 kB]
Get:32 http://archive.ubuntu.com/ubuntu bionic/main amd64 libbz2-dev amd64 1.0.6-8.1 [29.5 kB]
Get:33 http://archive.ubuntu.com/ubuntu bionic/main amd64 libexpat1-dev amd64 2.2.5-3 [122 kB]
Get:34 http://archive.ubuntu.com/ubuntu bionic/main amd64 zlib1g-dev amd64 1:1.2.11.dfsg-0ubuntu2 [176 kB]
Get:35 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpng-dev amd64 1.6.34-1ubuntu0.18.04.2 [177 kB]
Get:36 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfreetype6-dev amd64 2.8.1-2ubuntu2 [2539 kB]
Get:37 http://archive.ubuntu.com/ubuntu bionic/main amd64 pkg-config amd64 0.29.1-0ubuntu2 [45.0 kB]
Get:38 http://archive.ubuntu.com/ubuntu bionic/main amd64 libfontconfig1-dev amd64 2.12.6-0ubuntu2 [689 kB]
Get:39 http://archive.ubuntu.com/ubuntu bionic/main amd64 libgdbm-dev amd64 1.14.1-6 [79.3 kB]
Get:40 http://archive.ubuntu.com/ubuntu bionic/main amd64 xorg-sgml-doctools all 1:1.11-1 [12.9 kB]
Get:41 http://archive.ubuntu.com/ubuntu bionic/main amd64 x11proto-dev all 2018.4-4 [251 kB]
Get:42 http://archive.ubuntu.com/ubuntu bionic/main amd64 x11proto-core-dev all 2018.4-4 [2620 B]
Get:43 http://archive.ubuntu.com/ubuntu bionic/main amd64 libice-dev amd64 2:1.0.9-2 [46.8 kB]
Get:44 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libtinfo-dev amd64 6.1-1ubuntu1.18.04 [81.3 kB]
Get:45 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libncursesw5-dev amd64 6.1-1ubuntu1.18.04 [199 kB]
Get:46 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpthread-stubs0-dev amd64 0.3-4 [4068 B]
Get:47 http://archive.ubuntu.com/ubuntu bionic/main amd64 libreadline5 amd64 5.2+dfsg-3build1 [99.5 kB]
Get:48 http://archive.ubuntu.com/ubuntu bionic/main amd64 libsm-dev amd64 2:1.2.2-1 [16.2 kB]
Get:49 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libsqlite3-dev amd64 3.22.0-1ubuntu0.1 [633 kB]
Get:50 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libssl-dev amd64 1.1.1-1ubuntu2.1~18.04.3 [1566 kB]
Get:51 http://archive.ubuntu.com/ubuntu bionic/main amd64 libtcl8.6 amd64 8.6.8+dfsg-3 [881 kB]
Get:52 http://archive.ubuntu.com/ubuntu bionic/main amd64 libtk8.6 amd64 8.6.8-4 [693 kB]
Get:53 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxau-dev amd64 1:1.0.8-1 [11.1 kB]
Get:54 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxdmcp-dev amd64 1:1.1.2-3 [25.1 kB]
Get:55 http://archive.ubuntu.com/ubuntu bionic/main amd64 xtrans-dev all 1.3.5-1 [70.5 kB]
Get:56 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libxcb1-dev amd64 1.13-2~ubuntu18.04 [80.0 kB]
Get:57 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libx11-dev amd64 2:1.6.4-3ubuntu0.2 [640 kB]
Get:58 http://archive.ubuntu.com/ubuntu bionic/main amd64 x11proto-xext-dev all 2018.4-4 [2620 B]
Get:59 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxext-dev amd64 2:1.3.3-1 [82.1 kB]
Get:60 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxrender-dev amd64 1:0.9.10-1 [24.9 kB]
Get:61 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxft-dev amd64 2.3.2-1 [45.7 kB]
Get:62 http://archive.ubuntu.com/ubuntu bionic/main amd64 x11proto-scrnsaver-dev all 2018.4-4 [2628 B]
Get:63 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxss-dev amd64 1:1.2.2-1 [12.7 kB]
Get:64 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxt6 amd64 1:1.1.5-1 [160 kB]
Get:65 http://archive.ubuntu.com/ubuntu bionic/main amd64 libxt-dev amd64 1:1.1.5-1 [395 kB]
Get:66 http://archive.ubuntu.com/ubuntu bionic/main amd64 tcl8.6 amd64 8.6.8+dfsg-3 [14.4 kB]
Get:67 http://archive.ubuntu.com/ubuntu bionic/universe amd64 tcl amd64 8.6.0+9 [5146 B]
Get:68 http://archive.ubuntu.com/ubuntu bionic/main amd64 tcl8.6-dev amd64 8.6.8+dfsg-3 [887 kB]
Get:69 http://archive.ubuntu.com/ubuntu bionic/universe amd64 tcl-dev amd64 8.6.0+9 [5664 B]
Get:70 http://archive.ubuntu.com/ubuntu bionic/main amd64 tk8.6 amd64 8.6.8-4 [12.3 kB]
Get:71 http://archive.ubuntu.com/ubuntu bionic/universe amd64 tk amd64 8.6.0+9 [3178 B]
Get:72 http://archive.ubuntu.com/ubuntu bionic/main amd64 tk8.6-dev amd64 8.6.8-4 [679 kB]
Get:73 http://archive.ubuntu.com/ubuntu bionic/universe amd64 tk-dev amd64 8.6.0+9 [2964 B]
Get:74 http://archive.ubuntu.com/ubuntu bionic/main amd64 libffi-dev amd64 3.2.1-8 [156 kB]
Get:75 http://archive.ubuntu.com/ubuntu bionic/main amd64 libreadline-gplv2-dev amd64 5.2+dfsg-3build1 [122 kB]
Get:76 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 uuid-dev amd64 2.31.1-0.4ubuntu3.3 [33.2 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 18.5 MB in 2s (7685 kB/s)
Selecting previously unselected package multiarch-support.
(Reading database ... 11486 files and directories currently installed.)
Preparing to unpack .../multiarch-support_2.27-3ubuntu1_amd64.deb ...
Unpacking multiarch-support (2.27-3ubuntu1) ...
Setting up multiarch-support (2.27-3ubuntu1) ...
Selecting previously unselected package libxau6:amd64.
(Reading database ... 11489 files and directories currently installed.)
Preparing to unpack .../00-libxau6_1%3a1.0.8-1_amd64.deb ...
Unpacking libxau6:amd64 (1:1.0.8-1) ...
Selecting previously unselected package libbsd0:amd64.
Preparing to unpack .../01-libbsd0_0.8.7-1_amd64.deb ...
Unpacking libbsd0:amd64 (0.8.7-1) ...
Selecting previously unselected package libxdmcp6:amd64.
Preparing to unpack .../02-libxdmcp6_1%3a1.1.2-3_amd64.deb ...
Unpacking libxdmcp6:amd64 (1:1.1.2-3) ...
Selecting previously unselected package libxcb1:amd64.
Preparing to unpack .../03-libxcb1_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb1:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libx11-data.
Preparing to unpack .../04-libx11-data_2%3a1.6.4-3ubuntu0.2_all.deb ...
Unpacking libx11-data (2:1.6.4-3ubuntu0.2) ...
Selecting previously unselected package libx11-6:amd64.
Preparing to unpack .../05-libx11-6_2%3a1.6.4-3ubuntu0.2_amd64.deb ...
Unpacking libx11-6:amd64 (2:1.6.4-3ubuntu0.2) ...
Selecting previously unselected package libxext6:amd64.
Preparing to unpack .../06-libxext6_2%3a1.3.3-1_amd64.deb ...
Unpacking libxext6:amd64 (2:1.3.3-1) ...
Selecting previously unselected package x11-common.
Preparing to unpack .../07-x11-common_1%3a7.7+19ubuntu7.1_all.deb ...
dpkg-query: no packages found matching nux-tools
Unpacking x11-common (1:7.7+19ubuntu7.1) ...
Selecting previously unselected package libice6:amd64.
Preparing to unpack .../08-libice6_2%3a1.0.9-2_amd64.deb ...
Unpacking libice6:amd64 (2:1.0.9-2) ...
Selecting previously unselected package libsm6:amd64.
Preparing to unpack .../09-libsm6_2%3a1.2.2-1_amd64.deb ...
Unpacking libsm6:amd64 (2:1.2.2-1) ...
Selecting previously unselected package libexpat1:amd64.
Preparing to unpack .../10-libexpat1_2.2.5-3_amd64.deb ...
Unpacking libexpat1:amd64 (2.2.5-3) ...
Selecting previously unselected package libpng16-16:amd64.
Preparing to unpack .../11-libpng16-16_1.6.34-1ubuntu0.18.04.2_amd64.deb ...
Unpacking libpng16-16:amd64 (1.6.34-1ubuntu0.18.04.2) ...
Selecting previously unselected package libfreetype6:amd64.
Preparing to unpack .../12-libfreetype6_2.8.1-2ubuntu2_amd64.deb ...
Unpacking libfreetype6:amd64 (2.8.1-2ubuntu2) ...
Selecting previously unselected package ucf.
Preparing to unpack .../13-ucf_3.0038_all.deb ...
Moving old data out of the way
Unpacking ucf (3.0038) ...
Selecting previously unselected package fonts-dejavu-core.
Preparing to unpack .../14-fonts-dejavu-core_2.37-1_all.deb ...
Unpacking fonts-dejavu-core (2.37-1) ...
Selecting previously unselected package fontconfig-config.
Preparing to unpack .../15-fontconfig-config_2.12.6-0ubuntu2_all.deb ...
Unpacking fontconfig-config (2.12.6-0ubuntu2) ...
Selecting previously unselected package libfontconfig1:amd64.
Preparing to unpack .../16-libfontconfig1_2.12.6-0ubuntu2_amd64.deb ...
Unpacking libfontconfig1:amd64 (2.12.6-0ubuntu2) ...
Selecting previously unselected package libxrender1:amd64.
Preparing to unpack .../17-libxrender1_1%3a0.9.10-1_amd64.deb ...
Unpacking libxrender1:amd64 (1:0.9.10-1) ...
Selecting previously unselected package libxft2:amd64.
Preparing to unpack .../18-libxft2_2.3.2-1_amd64.deb ...
Unpacking libxft2:amd64 (2.3.2-1) ...
Selecting previously unselected package libxss1:amd64.
Preparing to unpack .../19-libxss1_1%3a1.2.2-1_amd64.deb ...
Unpacking libxss1:amd64 (1:1.2.2-1) ...
Selecting previously unselected package libmagic-mgc.
Preparing to unpack .../20-libmagic-mgc_1%3a5.32-2ubuntu0.2_amd64.deb ...
Unpacking libmagic-mgc (1:5.32-2ubuntu0.2) ...
Selecting previously unselected package libmagic1:amd64.
Preparing to unpack .../21-libmagic1_1%3a5.32-2ubuntu0.2_amd64.deb ...
Unpacking libmagic1:amd64 (1:5.32-2ubuntu0.2) ...
Selecting previously unselected package file.
Preparing to unpack .../22-file_1%3a5.32-2ubuntu0.2_amd64.deb ...
Unpacking file (1:5.32-2ubuntu0.2) ...
Selecting previously unselected package libglib2.0-0:amd64.
Preparing to unpack .../23-libglib2.0-0_2.56.4-0ubuntu0.18.04.3_amd64.deb ...
Unpacking libglib2.0-0:amd64 (2.56.4-0ubuntu0.18.04.3) ...
Preparing to unpack .../24-libsqlite3-0_3.22.0-1ubuntu0.1_amd64.deb ...
Unpacking libsqlite3-0:amd64 (3.22.0-1ubuntu0.1) over (3.22.0-1) ...
Preparing to unpack .../25-libssl1.1_1.1.1-1ubuntu2.1~18.04.3_amd64.deb ...
Unpacking libssl1.1:amd64 (1.1.1-1ubuntu2.1~18.04.3) over (1.1.0g-2ubuntu4.3) ...
Selecting previously unselected package tzdata.
Preparing to unpack .../26-tzdata_2019a-0ubuntu0.18.04_all.deb ...
Unpacking tzdata (2019a-0ubuntu0.18.04) ...
Selecting previously unselected package libpsl5:amd64.
Preparing to unpack .../27-libpsl5_0.19.1-5build1_amd64.deb ...
Unpacking libpsl5:amd64 (0.19.1-5build1) ...
Selecting previously unselected package wget.
Preparing to unpack .../28-wget_1.19.4-1ubuntu2.2_amd64.deb ...
Unpacking wget (1.19.4-1ubuntu2.2) ...
Selecting previously unselected package checkinstall.
Preparing to unpack .../29-checkinstall_1.6.2-4ubuntu2_amd64.deb ...
Unpacking checkinstall (1.6.2-4ubuntu2) ...
Selecting previously unselected package libbz2-dev:amd64.
Preparing to unpack .../30-libbz2-dev_1.0.6-8.1_amd64.deb ...
Unpacking libbz2-dev:amd64 (1.0.6-8.1) ...
Selecting previously unselected package libexpat1-dev:amd64.
Preparing to unpack .../31-libexpat1-dev_2.2.5-3_amd64.deb ...
Unpacking libexpat1-dev:amd64 (2.2.5-3) ...
Selecting previously unselected package zlib1g-dev:amd64.
Preparing to unpack .../32-zlib1g-dev_1%3a1.2.11.dfsg-0ubuntu2_amd64.deb ...
Unpacking zlib1g-dev:amd64 (1:1.2.11.dfsg-0ubuntu2) ...
Selecting previously unselected package libpng-dev:amd64.
Preparing to unpack .../33-libpng-dev_1.6.34-1ubuntu0.18.04.2_amd64.deb ...
Unpacking libpng-dev:amd64 (1.6.34-1ubuntu0.18.04.2) ...
Selecting previously unselected package libfreetype6-dev:amd64.
Preparing to unpack .../34-libfreetype6-dev_2.8.1-2ubuntu2_amd64.deb ...
Unpacking libfreetype6-dev:amd64 (2.8.1-2ubuntu2) ...
Selecting previously unselected package pkg-config.
Preparing to unpack .../35-pkg-config_0.29.1-0ubuntu2_amd64.deb ...
Unpacking pkg-config (0.29.1-0ubuntu2) ...
Selecting previously unselected package libfontconfig1-dev:amd64.
Preparing to unpack .../36-libfontconfig1-dev_2.12.6-0ubuntu2_amd64.deb ...
Unpacking libfontconfig1-dev:amd64 (2.12.6-0ubuntu2) ...
Selecting previously unselected package libgdbm-dev:amd64.
Preparing to unpack .../37-libgdbm-dev_1.14.1-6_amd64.deb ...
Unpacking libgdbm-dev:amd64 (1.14.1-6) ...
Selecting previously unselected package xorg-sgml-doctools.
Preparing to unpack .../38-xorg-sgml-doctools_1%3a1.11-1_all.deb ...
Unpacking xorg-sgml-doctools (1:1.11-1) ...
Selecting previously unselected package x11proto-dev.
Preparing to unpack .../39-x11proto-dev_2018.4-4_all.deb ...
Unpacking x11proto-dev (2018.4-4) ...
Selecting previously unselected package x11proto-core-dev.
Preparing to unpack .../40-x11proto-core-dev_2018.4-4_all.deb ...
Unpacking x11proto-core-dev (2018.4-4) ...
Selecting previously unselected package libice-dev:amd64.
Preparing to unpack .../41-libice-dev_2%3a1.0.9-2_amd64.deb ...
Unpacking libice-dev:amd64 (2:1.0.9-2) ...
Selecting previously unselected package libtinfo-dev:amd64.
Preparing to unpack .../42-libtinfo-dev_6.1-1ubuntu1.18.04_amd64.deb ...
Unpacking libtinfo-dev:amd64 (6.1-1ubuntu1.18.04) ...
Selecting previously unselected package libncursesw5-dev:amd64.
Preparing to unpack .../43-libncursesw5-dev_6.1-1ubuntu1.18.04_amd64.deb ...
Unpacking libncursesw5-dev:amd64 (6.1-1ubuntu1.18.04) ...
Selecting previously unselected package libpthread-stubs0-dev:amd64.
Preparing to unpack .../44-libpthread-stubs0-dev_0.3-4_amd64.deb ...
Unpacking libpthread-stubs0-dev:amd64 (0.3-4) ...
Selecting previously unselected package libreadline5:amd64.
Preparing to unpack .../45-libreadline5_5.2+dfsg-3build1_amd64.deb ...
Unpacking libreadline5:amd64 (5.2+dfsg-3build1) ...
Selecting previously unselected package libsm-dev:amd64.
Preparing to unpack .../46-libsm-dev_2%3a1.2.2-1_amd64.deb ...
Unpacking libsm-dev:amd64 (2:1.2.2-1) ...
Selecting previously unselected package libsqlite3-dev:amd64.
Preparing to unpack .../47-libsqlite3-dev_3.22.0-1ubuntu0.1_amd64.deb ...
Unpacking libsqlite3-dev:amd64 (3.22.0-1ubuntu0.1) ...
Selecting previously unselected package libssl-dev:amd64.
Preparing to unpack .../48-libssl-dev_1.1.1-1ubuntu2.1~18.04.3_amd64.deb ...
Unpacking libssl-dev:amd64 (1.1.1-1ubuntu2.1~18.04.3) ...
Selecting previously unselected package libtcl8.6:amd64.
Preparing to unpack .../49-libtcl8.6_8.6.8+dfsg-3_amd64.deb ...
Unpacking libtcl8.6:amd64 (8.6.8+dfsg-3) ...
Selecting previously unselected package libtk8.6:amd64.
Preparing to unpack .../50-libtk8.6_8.6.8-4_amd64.deb ...
Unpacking libtk8.6:amd64 (8.6.8-4) ...
Selecting previously unselected package libxau-dev:amd64.
Preparing to unpack .../51-libxau-dev_1%3a1.0.8-1_amd64.deb ...
Unpacking libxau-dev:amd64 (1:1.0.8-1) ...
Selecting previously unselected package libxdmcp-dev:amd64.
Preparing to unpack .../52-libxdmcp-dev_1%3a1.1.2-3_amd64.deb ...
Unpacking libxdmcp-dev:amd64 (1:1.1.2-3) ...
Selecting previously unselected package xtrans-dev.
Preparing to unpack .../53-xtrans-dev_1.3.5-1_all.deb ...
Unpacking xtrans-dev (1.3.5-1) ...
Selecting previously unselected package libxcb1-dev:amd64.
Preparing to unpack .../54-libxcb1-dev_1.13-2~ubuntu18.04_amd64.deb ...
Unpacking libxcb1-dev:amd64 (1.13-2~ubuntu18.04) ...
Selecting previously unselected package libx11-dev:amd64.
Preparing to unpack .../55-libx11-dev_2%3a1.6.4-3ubuntu0.2_amd64.deb ...
Unpacking libx11-dev:amd64 (2:1.6.4-3ubuntu0.2) ...
Selecting previously unselected package x11proto-xext-dev.
Preparing to unpack .../56-x11proto-xext-dev_2018.4-4_all.deb ...
Unpacking x11proto-xext-dev (2018.4-4) ...
Selecting previously unselected package libxext-dev:amd64.
Preparing to unpack .../57-libxext-dev_2%3a1.3.3-1_amd64.deb ...
Unpacking libxext-dev:amd64 (2:1.3.3-1) ...
Selecting previously unselected package libxrender-dev:amd64.
Preparing to unpack .../58-libxrender-dev_1%3a0.9.10-1_amd64.deb ...
Unpacking libxrender-dev:amd64 (1:0.9.10-1) ...
Selecting previously unselected package libxft-dev.
Preparing to unpack .../59-libxft-dev_2.3.2-1_amd64.deb ...
Unpacking libxft-dev (2.3.2-1) ...
Selecting previously unselected package x11proto-scrnsaver-dev.
Preparing to unpack .../60-x11proto-scrnsaver-dev_2018.4-4_all.deb ...
Unpacking x11proto-scrnsaver-dev (2018.4-4) ...
Selecting previously unselected package libxss-dev:amd64.
Preparing to unpack .../61-libxss-dev_1%3a1.2.2-1_amd64.deb ...
Unpacking libxss-dev:amd64 (1:1.2.2-1) ...
Selecting previously unselected package libxt6:amd64.
Preparing to unpack .../62-libxt6_1%3a1.1.5-1_amd64.deb ...
Unpacking libxt6:amd64 (1:1.1.5-1) ...
Selecting previously unselected package libxt-dev:amd64.
Preparing to unpack .../63-libxt-dev_1%3a1.1.5-1_amd64.deb ...
Unpacking libxt-dev:amd64 (1:1.1.5-1) ...
Selecting previously unselected package tcl8.6.
Preparing to unpack .../64-tcl8.6_8.6.8+dfsg-3_amd64.deb ...
Unpacking tcl8.6 (8.6.8+dfsg-3) ...
Selecting previously unselected package tcl.
Preparing to unpack .../65-tcl_8.6.0+9_amd64.deb ...
Unpacking tcl (8.6.0+9) ...
Selecting previously unselected package tcl8.6-dev:amd64.
Preparing to unpack .../66-tcl8.6-dev_8.6.8+dfsg-3_amd64.deb ...
Unpacking tcl8.6-dev:amd64 (8.6.8+dfsg-3) ...
Selecting previously unselected package tcl-dev:amd64.
Preparing to unpack .../67-tcl-dev_8.6.0+9_amd64.deb ...
Unpacking tcl-dev:amd64 (8.6.0+9) ...
Selecting previously unselected package tk8.6.
Preparing to unpack .../68-tk8.6_8.6.8-4_amd64.deb ...
Unpacking tk8.6 (8.6.8-4) ...
Selecting previously unselected package tk.
Preparing to unpack .../69-tk_8.6.0+9_amd64.deb ...
Unpacking tk (8.6.0+9) ...
Selecting previously unselected package tk8.6-dev:amd64.
Preparing to unpack .../70-tk8.6-dev_8.6.8-4_amd64.deb ...
Unpacking tk8.6-dev:amd64 (8.6.8-4) ...
Selecting previously unselected package tk-dev:amd64.
Preparing to unpack .../71-tk-dev_8.6.0+9_amd64.deb ...
Unpacking tk-dev:amd64 (8.6.0+9) ...
Selecting previously unselected package libffi-dev:amd64.
Preparing to unpack .../72-libffi-dev_3.2.1-8_amd64.deb ...
Unpacking libffi-dev:amd64 (3.2.1-8) ...
Selecting previously unselected package libreadline-gplv2-dev:amd64.
Preparing to unpack .../73-libreadline-gplv2-dev_5.2+dfsg-3build1_amd64.deb ...
Unpacking libreadline-gplv2-dev:amd64 (5.2+dfsg-3build1) ...
Selecting previously unselected package uuid-dev:amd64.
Preparing to unpack .../74-uuid-dev_2.31.1-0.4ubuntu3.3_amd64.deb ...
Unpacking uuid-dev:amd64 (2.31.1-0.4ubuntu3.3) ...
Setting up libexpat1:amd64 (2.2.5-3) ...
Setting up libbz2-dev:amd64 (1.0.6-8.1) ...
Setting up libpng16-16:amd64 (1.6.34-1ubuntu0.18.04.2) ...
Setting up libgdbm-dev:amd64 (1.14.1-6) ...
Setting up libpthread-stubs0-dev:amd64 (0.3-4) ...
Setting up fonts-dejavu-core (2.37-1) ...
Setting up libpsl5:amd64 (0.19.1-5build1) ...
Setting up tzdata (2019a-0ubuntu0.18.04) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Configuring tzdata
------------------

Please select the geographic area in which you live. Subsequent configuration
questions will narrow this down by presenting a list of cities, representing
the time zones in which they are located.

  1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
  2. America     5. Arctic     8. Europe    11. SystemV
  3. Antarctica  6. Asia       9. Indian    12. US
Geographic area:
```